site/pages/gc-subway-index.hbs

```diff
@@ -0,0 +1,106 @@
+---
+{
+"title": "GC-Subway Index",
+"language": "en",
+"altLangPrefix": "gc-subway-index",
+"breadcrumb": [
+{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+{ "title": "Business and industry", "link": "https://www.canada.ca/en/services/business.html" },
+],
+"tag": "GCWeb",
+"dateModified": "2020-11-16",
+"share": "true"
+}
+---
+<h1 property="name" id="wb-cont" class="provisional gc-thickline">GC-Subway Index
+</h1>
+<div class="row">
+	<div class="col-md-8">
+		<div class="provisional alert alert-info">
+			<h2>This is an optional Alert</h2>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+				ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+				ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+				reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+				sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+				est laborum.</p>
+		</div>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+			ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+			ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+			reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+			sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+			est laborum.</p>
+		<div class="panel panel-default brdr-rds-0">
+			<div class="panel-body">
+				<div class="row">
+					<div class="col-sm-5 col-lg-6">
+						<p class="mrgn-tp-sm">Text for Call-to-action button
+							Lorem ipsum dolor sit amet</p>
+					</div>
+					<div class="col-sm-7 col-lg-6">
+						<p><a href="#" class="btn btn-block btn-call-to-action">Call to action button</a></p>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<section class="provisional gc-subway">
+			<h2>Sections</h2>
+			<dl>
+				<dt>
+					<a href="#">
```

Comment:
I think it would actually be a good idea to change all the steps' `href` to "**demos/gc-subway/gc-subway-en.html**" so it shows the full template in action. What do you think @duboisp ?

Reply:
@namjohn920 Proceed with this change.